docs/building.rst

```diff
@@ -160,6 +160,41 @@ Install the ``acbuild`` binaries by downloading them from the
     sudo tar xzvf acbuild-v0.4.0.tar.gz -C tools --strip-components=1
     sudo chmod u+s tools/acbuild
 
+Alternatively to build the latest ``acbuild`` from master instead, start the
+container as before.
+
+::
+
+    sudo rkt run \
```

Comment:
Random question: Is sudo-less rkt on the roadmap? I feel like I've read something about this in the past.